src/Compute/Compute/help/New-AzVM.md

```diff
@@ -317,6 +318,38 @@ New-AzVM -ResourceGroupName $rgname -Location $loc -VM $vmConfig -DisableIntegri
 
 This example Creates a new VM with the GuestAttestation extension installed by default, then recreating the VM with DisableIntegrityMonitoring to prevent this.
 
+### Example 8: Creating a VM for Trusted Launch SecurityType, then enabling/diasbling SecureBoot and Vtpm flags.
+```powershell
+$rgname = "rgname";
+$loc = "eastus";
+ 
+New-AzResourceGroup -Name $rgname -Location $loc -Force;    
+
+# VM Profile & Hardware       
+$domainNameLabel1 = "d1" + $rgname;
+$vmsize = 'Standard_D4s_v3';
+$vmname1 = $rgname + 'V';
+$imageName = "Win2016DataCenterGenSecond";
+$disable = $false;
+$enable = $true;
+
+$password = Get-PasswordForVM;
```

Comment:
use just an insert like "<password>" and not the Get-Password cmdlet as it is only an internal cmdlet that customers do not have access to.

Reply:
good catch

Reply:
" <password > "